Docker image doesn't build: Golang out of date

When building the Docker image in the root `Dockerfile`, I encounter the following error:
```
go: downloading github.com/marten-seemann/qtls v0.2.3
go: extracting github.com/marten-seemann/qtls v0.2.3
go: downloading github.com/koron/go-ssdp v0.0.0-20180514024734-4a0ed625a78b
go: extracting github.com/koron/go-ssdp v0.0.0-20180514024734-4a0ed625a78b
go: extracting gonum.org/v1/gonum v0.6.0
go: extracting golang.org/x/text v0.3.2
# nhooyr.io/websocket
/go/pkg/mod/nhooyr.io/websocket@v1.7.4/conn.go:479:6: undefined: errors.Is
/go/pkg/mod/nhooyr.io/websocket@v1.7.4/conn.go:877:5: undefined: errors.As
/go/pkg/mod/nhooyr.io/websocket@v1.7.4/frame.go:262:5: undefined: errors.As
note: module requires Go 1.13
make: *** [build] Error 2
```

Examining the `Dockerfile`, I see the following:
```
FROM golang:1.12.4
...
```

PR inbound which applies the fix.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.12.4
+FROM golang:1.13.7
 LABEL maintainer="sparkle_pony_2000@qri.io"
 
 ADD . /go/src/github.com/qri-io/qri

